src/main/java/attendance/domain/Attendances.java

```diff
@@ -0,0 +1,67 @@
+package attendance.domain;
+
+import attendance.exception.AttendanceArgumentException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Attendances {
+    private final Map<LocalDate, Attendance> attendances = new HashMap<>();
+    private static final String DUPLICATE_ATTENDANCE_DATE = "이미 출석되었습니다. 수정 기능을 이용해주세요.";
+    private static final String NOT_EXIST_ATTENDANCE = "해당 날짜에 출석이 존재하지 않습니다.";
```

Comment:
```suggestion
    private static final String DUPLICATE_ATTENDANCE_DATE = "이미 출석되었습니다. 수정 기능을 이용해주세요.";
    private static final String NOT_EXIST_ATTENDANCE = "해당 날짜에 출석이 존재하지 않습니다.";
    private final Map<LocalDate, Attendance> attendances = new HashMap<>();
```

자바 컨벤션상 상수가 위에 위치하여야 합니다.

https://www.oracle.com/java/technologies/javase/codeconventions-fileorganization.html